src/main/resources/application-shared.properties

```diff
@@ -1,3 +1,4 @@
 # Header per traceid
 pn.log.trace-id-header=X-Amzn-Trace-Id
+pn.log.jti-id-header=x-pagopa-pn-jti
```

Comment:
@fabriziolana lo rinominerei in `jti-header`

Reply:
fatto